internal/output/json.go

```diff
@@ -92,8 +99,22 @@ func (f *JSON) Error(value interface{}) {
 
 	_, err = f.cfg.OutWriter.Write(b)
 	if err != nil {
-		multilog.Error("Could not write json output, error: %v", err)
+		if isPipeClosedError(err) {
+			logging.Error("Could not write json output, error: %v", err) // do not log to rollbar
+		} else {
+			multilog.Error("Could not write json output, error: %v", err)
+		}
+	}
+}
+
+func isPipeClosedError(err error) bool {
+	pipeErr := errors.Is(err, syscall.EPIPE)
```

Comment:
Should we also check for `io.ErrClosedPipe`?

Reply:
I don't think so, because the error message is "io: read/write on closed pipe" (https://pkg.go.dev/io#pkg-variables). That's not consistent with the rollbar error message we are seeing. I feel like we want to be as narrow as possible right now for this ticket since we're turning off rollbar logging in this case. If we start seeing this, then we can revisit.